src/store/Business/mutations.ts

```diff
@@ -0,0 +1,33 @@
+// import Vue from 'vue'
+import { BusinessIF } from '@/interfaces'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+
+export default {
+  // *** TODO: discuss this with Jonathan
+  setBusinessInfo (state: BusinessIF, val: BusinessIF) {
+    // state = val // doesn't work
+
+    // Vue.set(state, 'business', val) // doesn't work
+
+    Object.keys(val).forEach(key => {
+      // Vue.set(state, key, val[key]) // works but more complex than necessary
+      state[key] = val[key]
+    })
+  },
```

Comment:
After I suggested using `state = val` only to have you discover it doesn't work (sorry!), I think we should follow convention and use `state.businessInfo = val`.

Reply:
I'm doing more work in this area for the subject ticket. Can I change this in my next commit (so I can merge this PR asap)?

Reply:
I updated this in my next PR: https://github.com/bcgov/business-filings-ui/pull/461/commits/2873ac9eebd85d2e8a1c6c57b541e450b9daab86